starter_code/queries.md

```diff
@@ -4,80 +4,98 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+{name: "Babelgum"}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+{number_of_employees: {\$gt: 5000}}
+sort {number_of_employees: 1}
+limit 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+{ founded_year: { \$in: [ 2000, 2001, 2002, 2003, 2004, 2005] } }
```

Comment:
instead of writing all year you can write: Filter: {$and:[{founded_year: {$gte: 2000}},{founded_year: {$lte: 2005}}]}
it mean greater or equal than 2000 and less or equal than 2005